keyboards/keebio/viterbi/keymaps/ericgebhart/rules.mk

```diff
@@ -0,0 +1,24 @@
+# MCU name
```

Comment:
This file should only contain overrides, which it looks like there are none so should be deleted

Reply:
The default is rev2. My board is rev1.

Reply:
then you should build the correct revision, rather than what you have here. Leaving DEFAULT_FOLDER is still wrong in this case.

You should be invoking either,
`make keebio/viterbi/rev1:ericgebhart`
or
`qmk compile -kb=keebio/viterbi/rev1 -km=ericgebhart`